index.js

```diff
@@ -1,71 +1,92 @@
 'use strict';
-
 var chalk = require('chalk');
 var logUtil = require('./logUtil');
 var linesUtil = require('./linesUtil');
+var extend = require('util')._extend;

```

Comment:
Unfortunately, `util._extend()` is [deprecated](https://nodejs.org/api/util.html#util_util_extend_target_source). We've to use [`Object.assign()`](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_Objects/Object/assign) instead. Let's try that.